File: ServerApp/src/main/java/lk/earth/earthuniversity/report/entity/CountByMsjCity.java
package lk.earth.earthuniversity.report.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class CountByMsjCity {

    private Integer id;
    private String msjcity;
    private Long count;
    private double percentage;

    public CountByMsjCity() {  }

    public CountByMsjCity(String msjcity, Long count) {
        this.msjcity = msjcity;
        this.count = count;
    }

    public String getMsjcity() {
        return msjcity;
    }
    public void setMsjcity(String msjcity) {
        this.msjcity = msjcity;
    }
    public Long getCount() {
        return count;
    }
    public void setCount(Long count) {
        this.count = count;
    }
    public double getPercentage() {
        return percentage;
    }
    public void setPercentage(double percentage) {
        this.percentage = percentage;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Id
    public Integer getId() {
        return id;
    }

}

File: ClientApp/src/app/report/entity/countbymsjcity.ts
export class CountByMsjCity {

  public id !: number;
  public msjcity !: string;
  public count !: number;
  public perecentage !: number;

  constructor(id:number,msjcity:string,count:number,perecentage:number) {
    this.id=id;
    this.msjcity=msjcity;
    this.count=count;
    this.perecentage=perecentage;
  }

}
